test/validators/belongs_to_host_taxonomy_validator_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class Validatable
```

Comment:
If other test files define the same class, it may cause clashes. What do you think about having this nested inside the test class?